bin/make-snake

```diff
@@ -96,6 +96,17 @@ function colourEcho
     colour_echo "$1"
 }
 
+function ensureContentDir
+{
+    # ensure that the content dir exists and it contains the default theme
+    for f in ~/Snake-content/*.xml; do
```

Comment:
if Snake-content isn't there, it still triggers and creates the dir + copies the file